backend/EasyMeets.Core/EasyMeets.Core.WebAPI/service-account-file.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "service_account",
+  "project_id": "easymeets-4ed60",
+  "private_key_id": "5c4f4c464f15e8e4ec2155870590b7faedb57d90",
+  "client_email": "firebase-adminsdk-ud2yz@easymeets-4ed60.iam.gserviceaccount.com",
+  "client_id": "111694983450380510761",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-ud2yz%40easymeets-4ed60.iam.gserviceaccount.com"
+}
```

Comment:
Please move private items to environment variables

Reply:
And delete this file
Not security to appsetting file
Secure to environment variables